.github/workflows/ios.yml

```diff
@@ -0,0 +1,23 @@
+name: iOS
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build and Test FlipperZero(ios) scheme using iPhone 11 simulator
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Xcode build
+        run: |
+          sudo xcode-select --switch /Applications/Xcode_12.app
+          xcodebuild -sdk iphoneos -workspace ./FlipperZero/FlipperZero.xcodeproj/project.xcworkspace -scheme 'FlipperZero(iOS)'  clean archive -archivePath ./output/build/archive/FlipperZeroiOS.xcarchive CODE_SIGNING_ALLOWED=NO
```

Comment:
I think right now we don't need to archive on each PR update.
Later we might add archiving and uploading of the package on each push to `master`.

Reply:
Good point